_pages/docs/how-does-it-work.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: inner
+title: How Does it Work?
+lead_text: ''
+permalink: /docs/how-does-it-work/
+---
+
+# How Does it Work?
+
+Smart Mastering consists of configuration-driven matching and merging. While
+the [How To Use][how-to-use] page will show you how to access this
+functionality, this page describes the matching and merging processes that
+make up Smart Mastering.
+
+The Smart Mastering process focuses on one document at a time, identifying
+any other documents that appear to represent the same entity (Matching) and
+then combining the values in those documents to create a new document (Merging).
+
+## Collections
+
+Smart Mastering Core uses collections to separate types of content. Applications
+should use [`constants.xqy`][constants] for the names of the collections. The
```

Comment:
All this talk of constants might just confuse readers. It's only relevant if you are writing xqy or sjs. REST users need not know about constants.

Reply:
If you're using REST, you still need to make sure your harmonized content is in the CONTENT collection, and your search options need to be set up to only look there.